packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/nix/changeEveryIfState.yml

```diff
@@ -0,0 +1,37 @@
+languageId: nix
+command:
+  version: 6
+  spokenForm: change every if state
+  action:
+    name: clearAndSetSelection
+    target:
+      type: primitive
+      modifiers:
+        - type: everyScope
+          scopeType: {type: ifStatement}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |
+    {
+      key = if a then b else c;
```

Comment:
are all `if`s just inline expressions, or are there standalone `if` statements? This code looks like a ternary, which we don't match as an `if` statement in typescript, but it's worth looking at what we do in Rust, where the ternaries look more like this syntactically

Reply:
Ya this is the only form of if in nix, always part of an assignment. The reason I was doing it this way is because, especially when people break up the assignment expressions into multi-line (which seems quite common), it "feels" like a normal if statement more than a ternary? But I'll also check how rust does it ya.

Not sure if @auscompgeek you have thoughts since you seem familiar with the nix language?

Reply:
Yeah, everything's an expression in Nix. I'd lean towards what we do in Haskell or Rust.

Reply:
Playing with Rust it seems like what I've done for nix is pretty much the same. Although the rust nested if behavior seems a bit buggy/weird to me, so there are a few inconsistencies. Can discuss the rust questions here or in a separate thread if you prefer.

Reply:
I already forget what the weird rust behavior I had wanted to mention was, so marking this as resolved since as is nix works the same as rust.